src/capi.rs

```diff
@@ -90,6 +90,21 @@ pub enum EncoderStatus {
   NotReady = -2,
 }
 
+impl EncoderStatus {
+  fn to_c(&self) -> *const u8 {
+    use self::EncoderStatus::*;
+    match self {
+      Success => "Normal operation\0".as_ptr(),
+      NeedMoreData => "The encoder needs more data to produce an output packet\0".as_ptr(),
+      EnoughData => "There are enough frames in the queue\0".as_ptr(),
+      LimitReached => "The encoder has already produced the number of frames requested\0".as_ptr(),
+      Encoded => "A Frame had been encoded but not emitted yet\0".as_ptr(),
+      Failure => "Generic fatal error\0".as_ptr(),
+      NotReady => "First-pass stats data not retrieved or not enough second-pass data provided\0".as_ptr(),
```

Comment:
Is there a way to add a test for these to make sure the Tust and C APIs do not diverge?

Reply:
Currently it would not compile.